extensions/measurement-tracking/src/panels/PanelMeasurementTableTracking/index.js

```diff
@@ -148,17 +148,14 @@ function PanelMeasurementTableTracking({ servicesManager, extensionManager }) {
   }
 
   function exportReport() {
-    const dataSources = extensionManager.getDataSources();
-    const dataSource = dataSources[0];
     const measurements = MeasurementService.getMeasurements();
     const trackedMeasurements = measurements.filter(
       m =>
         trackedStudy === m.referenceStudyUID &&
         trackedSeries.includes(m.referenceSeriesUID)
     );
 
-    // TODO -> local download.
-    DICOMSR.downloadReport(trackedMeasurements, dataSource);
+    DICOMSR.downloadReport(trackedMeasurements, ['ArrowAnnotate']);
```

Comment:
Can we make this more obvious?

`const additionalFindings = ['ArrowAnnotate'];`

and then pass that in, so it's a named param.

Reply:
gotcha.